src/workerd/api/r2-admin.c++

```diff
@@ -13,8 +13,15 @@
 
 namespace workerd::api::public_beta {
 jsg::Ref<R2Bucket> R2Admin::get(jsg::Lock& js, kj::String bucketName) {
-  return jsg::alloc<R2Bucket>(featureFlags, subrequestChannel, kj::mv(bucketName),
-      R2Bucket::friend_tag_t{});
+  KJ_IF_MAYBE(a, this->adminAccount) {
+    KJ_IF_MAYBE(j, this->jwt) {
+      return jsg::alloc<R2Bucket>(featureFlags, subrequestChannel, kj::mv(bucketName),
+        kj::mv(*a), kj::mv(*j), R2Bucket::friend_tag_t{});
```

Comment:
This pattern is going to be problematic. Specifically, the `kj::mv(*a)` and `kj::mv(*j)` here will move away the values of `this->adminAccount` and `this->jwt` without clearing the corresponding `kj::Maybe`s. If this method is called twice you're going to have issues. If you want this to be a one-time-use method, make sure to reset both `this->adminAccount` and `this->jwt` to `nullptr` explicitly after the moves. If you're meant to be able to call `get` multiple times, however, the `kj::mv(...)`s here are not appropriate.

Reply:
This should not be a one-time-use, so we'll need to fix the `kj::mv` here.

Reply:
@OilyLime heres an example of one way to make a copy of the member variable (adminBucket), and then `kj::mv` the copy (bucket): https://github.com/cloudflare/workerd/blob/main/src/workerd/api/r2-bucket.c++#L363-L364.

@jasnell might have a better or preferred suggestion for how to fix it.

Reply:
Is `kj::heapString(*a)` more idiomatic than `kj::str(*a)`?

Reply:
Ah, no, I find myself defaulting to it for some reason but you should just stick with `kj::str(*a)` since it works in more cases.